packages/website/pages/index.js

```diff
@@ -23,13 +24,17 @@ export default function Home() {
 
   return (
     <>
+      {shouldShowSunsetAnnouncement() && (
+        <PageBannerPortal.PageBanner>
+          <W3upMigrationRecommendationCopy />
+        </PageBannerPortal.PageBanner>
+      )}
+
       <main className="page page-index">
         {sections.map((section, index) => (
           <BlockBuilder id={`section_${index + 1}`} key={`section_${index + 1}`} subsections={section} />
         ))}
       </main>
-
-      <Scroll2Top />
```

Comment:
When this was present, it was weirdly somehow resulting in an element from inside the scroll2top component getting mixed up with the PageBanner children. It happened in firefox and chrome, and I have no idea how.... The scroll2top component code seems relatively innocuous so I don't know why it would affect dom manip. The only way to fix was to not use the component here. I know its dumb, but without this, the bug in the dom that react rendered made the home page banner not render at all.

workaround: remove this Scroll2Top for now. We can add it back in later if we really need it, but we probably don't, and this whole website is being sunset anyway!